src/scss/mega/articles.scss

```diff
@@ -1,18 +0,0 @@
-@if $o-header-is-silent == false {
```

Comment:
I think this is fine though, I can't foresee us taking on any meganav work for a while and it would only be a small change if and when we do pick it up.